infrastructure/docker/services/builder/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN apk add --no-cache \
 
 RUN pip install pycodestyle
 
-RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+COPY --from=composer:1.9.3 /usr/bin/composer /usr/bin/composer
```

Comment:
Does that mean that the version is fixed? So it will never be updated and everybody will stay on old version until they notice they have to change the version here? Not really good for DX IMO

Reply:
> Does that mean that the version is fixed

Yes

> So it will never be updated and everybody will stay on old version until they notice they have 
to change the version here? 

Yes, and:
* It's already the case: until you destroy your base image, you keep the same version
* Before this patch you got an un-predictable version as state in the issue
* Now it's much better; This is how Ops works: you fix a version and you stick on it. The infra really depends on the code: It's predictable and it does not depends on time or when you install a software
* This is what we do with: docker-composer (pip env), node, php, alpine version etc.

Reply:
It means if someone updates the composer image version here, docker-compose will rebuild the image and the container. So everyone will be up to date. Right?

Reply:
> It means if someone updates the composer image version here, docker-compose will rebuild the image and the container. So everyone will be up to date. Right?

Wep :+1:

Reply:
> Now it's much better; This is how Ops works: you fix a version and you stick on it. The infra really depends on the code: It's predictable and it does not depends on time or when you install a software
I agree on the concept BUT in reality we never fix software versions in most of our production (Nginx, Rabbitmq, PHP patch version, etc). For a stack to be mostly used for dev env purposes I still find this quite useful.

In this particular case, we rarely need to update Composer (as for nginx and so on). I'm used to delete image and container for my old project, so I will always rebuild the project from scratch if needed. And anyway, it's not so complicated to destroy builder image.

I'm just afraid that most of people will stay on an old Composer version from now on, even if they completly rebuild their project's builder (think fe of a security issue found on Composer).

But anyway let's merge this PR for now because the "auto" rebuild for everyone is still interesting from a DX perspective.